crypto/conf/conf_api.c

```diff
@@ -17,6 +15,7 @@
 #include <openssl/conf_api.h>
 
 DEFINE_STACK_OF(CONF_VALUE)
+DEFINE_LHASH_OF(CONF_VALUE);
```

Comment:
There is an inconsistency here between LHASH and STACK. In the former you don't need to add a ";" to the end of the `DEFINE_STACK_OF()` line. However with `DEFINE_LHASH_OF()` you must do so. They should probably work the same way.

Reply:
okay DEFINE_LHASH_OF does not need the trailing semicolon now.